Templates/bootstrap4_template.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
```

Comment:
Could you add a one-line comment at the top explaining why one would choose to use this template? That is, what benefits does it provide?